SU-SWS/stanford_basic
# D8CORE-2189: Update to Decanter 6.0.6

# READY FOR REVIEW 

# Summary
- Updates build to Decanter 6.0.6

# Review By (Date)
- June 11th EOD

# Urgency
- Medium

# Steps to Test

1. Review code for any irregularities or oddities
2. Compare change you see in the code to the Decanter release notes: https://github.com/SU-SWS/decanter/releases/tag/6.0.6 to ensure that nothing extra got in or was dropped.

# Affected Projects or Products
- Everything

# Associated Issues and/or People
- D8CORE-2189

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
